packages/plugins/web3/src/index.js

```diff
@@ -52,11 +51,59 @@ class EmbarkWeb3 {
   }
 
   async registerInVm(params, cb) {
-    this.events.request("embarkjs:contract:runInVm", params.contract, cb);
+    const contract = params.contract;
+    const abi = contract.abiDefinition;
+    const gasLimit = 6000000;
```

Comment:
Should `gasLimit = 6000000` be defined in `embark-core/constants`?

Reply:
I moved to a constant, but not to embark-core/constants, reason is that  this file will disapear, and this code is old existing code and I rather not risk breaking it more if we are going to remove this whole module soon anyway